app/services/article_creation_service.rb

```diff
@@ -22,6 +22,7 @@ def create!
     article.collection = Collection.find_series(series, user) if series.present?
 
     if article.save
+      NotificationSubscription.create(user: user, notifiable_id: article.id, notifiable_type: "Article", config: "all_comments")
```

Comment:
what happens if `NotificationSubscription.create` fails?

Reply:
Nothing! 😅 I'm assuming that if a user is creating an article for the first time, they shouldn't be subscribed to the new article. Should be fine right?

Reply:
ohh right :D sorry